docs/api/protocol.md

```diff
@@ -285,6 +346,24 @@ which sends a `Buffer` as a response.
 Intercepts `scheme` protocol and uses `handler` as the protocol's new handler
 which sends a new HTTP request as a response.
 
+### `protocol.interceptStreamProtocol(scheme, handler[, completion])`
+
+* `scheme` String
+* `handler` Function
+  * `request` Object
+    * `url` String
+    * `headers` Object
+    * `referrer` String
+    * `method` String
+    * `uploadData` [UploadData[]](structures/upload-data.md)
+  * `callback` Function
+    * `stream` (Stream | [StreamProtocolResponse](structures/stream-protocol-response.md)) (optional)
```

Comment:
I'm assuming this is a `ReadableStream`, if so can we be explicit so I can add catches for streams in the typescript converter (we don't currently handle Streams)?

Reply:
:+1: